src/utils/types/ApplicationCommands.ts

```diff
@@ -47,12 +81,16 @@ interface ApplicationCommandOptions {
 	min_value?: number;
 	max_value?: number;
 	autocomplete?: boolean;
+	name_localizations?: Localization;
+	description_localizations?: Localization;
 }
 
 export interface ApplicationCommand {
```

Comment:
Why do we have both an `interface` and a `class` for ApplicationCommand? Seems kinda redundant. Also, be aware that declaration merging will occur when declaring multiple types with the same name (`class` declarations emit a type definition implicitly)

Reply:
This was added before the class was created, can prob safely be removed unless some other parts need it.